docs/snapshotting/versioning.md

```diff
@@ -3,13 +3,27 @@
 This document describes how Firecracker persists its state across multiple versions, diving deep into the snapshot format, encoding, compatibility and limitations. 
 
 ## Introduction
-The design behind the snapshot implementation enables version tolerant save and restore across multiple Firecracker versions which we call a version space. For example, one can pause a microVM, save it to disk with Firecracker version **0.26.0** and later load it in Firecracker version **0.24.0**. It also works  in reverse: Firecracker version **0.26.0** loads what  **0.24.0** saves. 
+The design behind the snapshot implementation enables version tolerant save and restore across multiple Firecracker versions which we call a version space. For example, one can pause a microVM, save it to disk with Firecracker version **0.23.0** and later load it in Firecracker version **0.24.0**. It also works  in reverse: Firecracker version **0.23.0** loads what  **0.24.0** saves. 
 
-Below is an example graph showing backward and forward snapshot compatibility for a version space formally represented as `[v2; vn]`. This is the general picture, but keep in mind that future corner cases might remove edges from this graph.
+Below is an example graph showing backward and forward snapshot compatibility. This is the general picture, but keep in mind that when adding new features some version translations would not be possible.
```

Comment:
I think, as formulated now, this needs examples.

Reply:
Will add that with v0.25 where this scenario is clear: you cannot save the snapshot in v0.23 format for a microvm configured with balloon emulation.